admin/src/react-components/system-editor.js

```diff
@@ -334,6 +335,28 @@ class SystemEditorComponent extends Component {
             </ListItem>
           </List>
         </Card>
+        <Card className={this.props.classes.container}>
+          <Title title="Hubs Cloud" />
+          <CardContent className={this.props.classes.info}>
+            <Typography variant="title" gutterBottom>
+              Your hub version:
+            </Typography>
+            {configs.IS_LOCAL_OR_CUSTOM_CLIENT ? (
+              <>
+                <Typography variant="body1" gutterBottom>
+                  {`App client: Custom client`}
+                </Typography>
+                <Typography variant="body1" gutterBottom>
+                  {`(Undeploy custom client to run build v${process.env.BUILD_VERSION || "?"})`}
```

Comment:
nit: remove the "v" for consistency
```suggestion
                  {`(Undeploy custom client to run build ${process.env.BUILD_VERSION || "?"})`}
```